yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1462,7 +1462,8 @@ func considerQueryForIssueDetection(collectedSchemaList []string) bool {
 }
 
 const (
-	ORACLE_PARTITION_DEFAULT_COLOCATION = `For sharding/colocation recommendations, each partition is treated individually. During the export schema phase, all the partitions of a partitioned table are currently created as colocated by default. 
+	COLOCATED_TABLE_RECOMMENDATION_CAVEAT = `If there are any tables that receive disproportionately high load, ensure that they are NOT colocated to avoid the colocated tablet becoming a hotspot.`
```

Comment:
Couple of points: 
1. Can we additionally mention: "See here for more considerations for colcoated tables - https://docs.yugabyte.com/preview/explore/colocation/#limitations-and-considerations"
2. Do you think it would be better to put this statically in the HTML report (near the colocated tables list)? Ofc the problem with that is that it won't be present in the json report..